packages/push/src/push_vm/stack.rs

```diff
@@ -261,10 +261,13 @@ impl<T> Stack<T> {
     /// Returns `StackError::Underflow` error if the stack has less than
     /// two elements.
     pub fn top2(&self) -> Result<(&T, &T), StackError> {
-        let index_second_to_top = self.size().checked_sub(2).ok_or(StackError::Underflow {
-            num_requested: 2,
-            num_present: self.size(),
-        })?;
+        let index_second_to_top =
+            self.size()
+                .checked_sub(2)
+                .ok_or_else(|| StackError::Underflow {
+                    num_requested: 2,
+                    num_present: self.size(),
+                })?;
```

Comment:
This change was made to address a new Clippy warning in nightly. Since there's a function call (`self.size()`) embedded in the expression, the warning recommends using `.ok_or_else()` to make the closure lazy, and potentially avoid that function call when it isn't necessary.

Reply:
Seems Good :+1: